data-raw/algorithm.csv

```diff
@@ -0,0 +1,3 @@
+name,logic
+hba1c,(analysiscode == 'NPU27300' AND value >= 48) OR (analysiscode == 'NPU03835' AND value >= 6.5)
```

Comment:
This is how I was thinking the logic would be stored and written down.

Reply:
Very clever! For HbA1c we (currently) convert NPU03835 (%) values to NPU27300 (mmol/mol) values in the pipeline, so we could potentially simplify the logic to ```hba1c,(analysiscode %in% c('NPU27300', 'NPU03835') AND value >= 48)```

Reply:
I think it might be faster from a performance point of view to filter only and not do an additional calculation step before hand.